stl/inc/experimental/algorithm

```diff
@@ -0,0 +1,45 @@
+// algorithm experimental header
+
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#pragma once
+#ifndef _EXPERIMENTAL_ALGORITHM_
+#define _EXPERIMENTAL_ALGORITHM_
+#include <yvals_core.h>
+#if _STL_COMPILER_PREPROCESSOR
+
+#include <experimental/random>

```

Comment:
WG21-N4840 requires `<experimental/algorithm>` to include `<algorithm>`.

Reply:
So if the paper has an `#include` in the synopsis then the implementation must have it as well even if it's pulled in through another header (`<random>` in this case )?

Reply:
Oh, I forgot about the transitive include. Then it’s not necessary (the behavior is the same either way and that’s all the Standard/TS cares about), but I’d still prefer to see it because that makes the code robust to header inclusion refactoring. 😸